xbmc/interfaces/legacy/ModuleXbmc.h

```diff
@@ -971,6 +971,38 @@ namespace XBMCAddon
 #else
     String convertLanguage(const char* language, int format);
 #endif
+
+#ifdef DOXYGEN_SHOULD_USE_THIS
+    ///
+    /// \ingroup python_xbmc
+    /// @brief \python_func{ xbmc.getOsName() }
+    ///-----------------------------------------------------------------------
+    /// @brief Returns Kodi's operating system name
+    ///
+    /// @return operation system name
+    ///
+    /// ------------------------------------------------------------------------
+    ///
+    /// example output: OS X
+    ///
+    getOsName();
+#else
+    String getOsName();
+#endif
+
+#ifdef DOXYGEN_SHOULD_USE_THIS
+    ///
+    /// \ingroup python_xbmc
+    /// @brief \python_func{ xbmc.getSystemPropertyString(propertyName) }
+    ///-----------------------------------------------------------------------
+    /// @brief Returns an Os dependend system property
+    ///
+    /// @return value assigned to propertyName
+    ///
```

Comment:
could you add a few examples here?
it's not clear to me what 'system properties' are...

also, is this android only?